falcon/request.py

```diff
@@ -1160,6 +1162,8 @@ class RequestOptions(object):
             request's content type is
             *application/x-www-form-urlencoded* (default ``False``). In
             this case, the request's content stream will be left at EOF.
+        auto_parse_query_string_lists: Set to ``False`` in order to disable
```

Comment:
I can not think of a good and short description of a use case except "If you need it for some reason".
My use case was that it was just easier to pass the values this way in the JS framework I use. It also seemed more "standard".